Userland/Libraries/LibWasm/AbstractMachine/Interpreter.cpp

```diff
@@ -437,7 +437,10 @@ void BytecodeInterpreter::interpret(Configuration& configuration, InstructionPoi
         for (size_t i = 0; i < frame.arity(); ++i)
             results.prepend(configuration.stack().pop());
         // drop all locals
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
         Optional<Label> last_label;
+#pragma GCC diagnostic pop
```

Comment:
Hmm this is suspicious.

Looking at Lablel... it looks like it holds a distinct ordered ID for InstructionPointer? Are those default initialized to 0? Or is the m_instruction_pointer of the label possibly what's causing it to cry here? I swear there must be tests of default initialized Optionals...

Reply:
Wait I just made that comment, how is it outdated? Is Github's cache messing with me? :(

Reply:
It's not at all about it being default initialised, it seems to be finding _some_ path that leaves this thing unintialised, I wasn't able to figure out what that is, but it's not through any normal paths; I took a look at the overly-huge tree dump but nothing interesting caught my eye

Reply:
> Wait I just made that comment, how is it outdated? Is Github's cache messing with me? :(

The whole thing is removed in 594fc0783160c70a74b6d02000b95fe3365ae16c.